### ISSUE TITLE
Archiving error when build .xcodeproj

### ISSUE THREAD

**Issue by cookle** [gurhub/surmagic#16]
First of all, forgive my poor english

## Describe the bug
i got an error when i build a framework

## To Reproduce
1. change Surfile file like below
```
<dict>
      <key>sdk</key>
      <string>xxxxx</string>
      <key>project</key>
      <string>xxxxx.xcodeproj</string>
      <key>scheme</key>
      <string>xxxxx</string>
</dict>
```
2. run surmagic xcf
3. got an error: 
 ["xcodebuild", "-quiet", "archive", "-project xxxxx.xcodeproj", "xxxxx.xcodeproj", "-sdk", "macosx", "-scheme", "xxxxx", "-archivePath", "./build/macOSCatalyst.xcarchive", "SKIP_INSTALL=NO", "SUPPORTS_MACCATALYST=YES"])
**xcodebuild: error: invalid option '-project xxxxx.xcodeproj'**

I think "-project xxxxx.xcodeproj" is the reason why i got this error and #14 means the same as me

## Details
  * macOS version : Monterey 12.4 (21F79)
  * Swift version : 5.5
  * Your CPU type : Intel

## Checklist
- [❌] If possible, I've reproduced the issue using the `master` branch of this package
- [✅] I've searched for [existing GitHub issues](https://github.com/gurhub/surmagic/issues)

**Comment by gurhub** [comment]
Hi @cookle, 

thank you for contributing. Yes, I see the issue, it's probably a bug, I don't have time to test right now. You can safely use with workspace option for now. Even if you don't need you can convert your project to a workspace as a workaround if you really need to use it right now. But I'll try to fix the error as soon as possible on the weekends. 

Best

**Comment by gurhub** [comment]
Thank you for your effort. Fixed on [version 1.2.7.](https://github.com/gurhub/surmagic/releases/tag/v1.2.7)
